src/main/proto/netflix/titus/titus_vpc_api.proto

```diff
@@ -37,10 +37,17 @@ message GetAllocationResponse {
   SignedAddressAllocation signedAddressAllocation = 2;
 };
 
+message DeleteAddressRequest {
+  string uuid = 1;
+}
+
+message DeleteAddressResponse {}
+
 service UserIPService {
   // Static IP Address flow
   rpc AllocateAddress(AllocateAddressRequest) returns (AllocateAddressResponse);
   rpc GetAllocation(GetAllocationRequest) returns (GetAllocationResponse);
+  rpc DeleteAddress(DeleteAddressRequest) returns (DeleteAddressResponse);
```

Comment:
I'm still not 100% sure about the terms for this, but does this actually delete addresses? Or will it actually delete *Allocations* ?

Maybe it would be better named `DeallocateAddress` as a mirror of AllocateAddress ?

Reply:
So this would, 
1. delete the allocation from the vpc service
2. delete the v4 and v6 SCRs which would release the addresses.

We could mirror it like `DeallocateAddress` from the vpc service context.

Reply:
It's kinda up to you all.
Making a good api is kinda an art.

Reply:
I like Deallocate. Lets change to that.